src/display/annotation_layer.js

```diff
@@ -459,6 +459,9 @@ class AnnotationElement {
    * @memberof AnnotationElement
    */
   _createPopup(trigger, data) {
+    // we dont want to render any of the floating popup for annotation metadata
+    return;
```

Comment:
Personal preference. Since we are returning out of this early, could you comment out the code that's in this method? All good if you prefer to not have it commented out.

Reply:
Seems reasonable to me. I used to err on the side of "delete commented out code, since git will remember it"... but git remembering it doesnt mean anybody else does!